nilearn/datasets/atlas.py

```diff
@@ -327,14 +357,22 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, keys are:
 
-        - "maps": nibabel.Nifti1Image, 4D maps if a probabilistic atlas is
-          requested and 3D labels if a maximum probabilistic atlas was
-          requested.
+            - 'maps': :class:`~nibabel.nifti1.Nifti1Image`, 4D maps if a
+              probabilistic atlas is requested, and 3D labels if a maximum
+              probabilistic atlas was requested.
+
+                .. note::
 
-        - "labels": string list, labels of the regions in the atlas.
+                    For some atlases, it can be the case that some regions
+                    are no longer present. In this case, the integers in the
+                    3D parcellation image might not be consecutive.
```

Comment:
But we're heading toward another heuristic, right ?

Reply:
I'm not really sure which heuristic you are referring to. We are indeed planning to add more convenient mappings as dictionaries `label -> integer_value` and `integer_value -> label`, but the image can still have non consecutive integer values when some regions are removed by thresholding or resampling for instance. 
I think that the addition of a small note in the documentation saying that this can happen and that it is not a bug would reassure users facing this situation.